operations/exports.md

```diff
@@ -0,0 +1,158 @@
+                    "SizeInBytes": 1215279
+                },
+                {
+                    "Url": "https://example.com/exports/3fa85f64-5717-4562-b3fd-2c963f66afa6-2.jsonl?example=signature",
+                    "SizeInBytes": 1398362
+                },
+            ]
+        },
+        {
+            "Id": "09708665-0e31-4b23-b337-b0a000be0df0",
+            "Status": "Pending",
+            "EntityType": "OrderItem",
+            "Files": [],
+            "ExpiresUtc": null
+        },
+        {
+            "Id": "706dc6d5-9511-4751-825e-538ce99da2ce",
+            "Status": "Processing",
+            "EntityType": "OrderItem",
+            "Files": [],
+            "ExpiresUtc": null
+        },
+        {
+            "Id": "706dc6d5-9511-4751-825e-538ce99da2ce",
+            "Status": "Expired",
+            "EntityType": "OrderItem",
+            "Files": [],
+            "ExpiresUtc": "2023-10-24T14:12:30Z"
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Exports` | array of [Exports](#export) | required | Requested exports. |
+
+#### Export
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the export. |
+| `Status` | string [Export status](#export-status) | required | Current status of the export. |
+| `EntityType` | string [Entity type](#entity-type) | required | Type of exported entities. |
+| `Files` | array of [Exported files](#exported-file) | required | Files with exported data, if available. |
+| `ExpiresUtc` | string | optional | Expiration date and time of the export in UTC timezone in ISO 8601 format. After this time the [exported files](#exported-file) may no longer be available for download. |
+
+#### Export status
+
+* `Pending` - export was created and didn't start yet
+* `Processing` - export process has started
+* `Success` - export has been successfully finished and [exported files](#exported-file) are available for download
+* `Expired` - export has expired and exported files are no longer available
+* `Failed` - export has failed and exported files are not available; you can try creating a new export
+
+#### Exported file
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Url` | string | required | URL of the exported file for download in [JSON Lines](https://jsonlines.org/) format. |
+| `SizeInBytes` | number | required | Total size of the exported file in bytes. |
+
+## Add export
+
+Create a pending export. Export will include all entities of the specified `EntityType` from all enterprises accessible for the integration (if [Portfolio Access Tokens](../guidelines/multi-property.md) is used).
+
+If there is already an export with status `Pending` or `Processing` for the specified entity type, request for new export will be rejected.
```

Comment:
I would probably include this in the first sentance that describes this operation?

```suggestion
Create a pending export unless there is already an export with status `Pending` or `Processing` for the specified entity type in which case the request will be rejected.
```

Reply:
Also I would mention, what happens when there is export that is done and I start a new one - if it's gets deleted or not? 🤔 I can imagine I would be interested in such information as a developer.

Reply:
I see your point but I don't agree about putting it into the first sentence. In general, I think it's preferred to start with basic situation (= there are 0 existing exports) and then dive into details like this one. I agree about your second comment though.

Reply:
Added the following sentence:

> Successful exports remain available for download until the expiration date specified in `ExpiresUtc`.